zen/Dockerfile

```diff
@@ -1,32 +1,27 @@
-FROM skilldlabs/frontend:latest
+FROM mhart/alpine-node:7
 
 MAINTAINER Andy Postnikov
 
-ENV NPM_PACKAGES "bower \
-  breakpoint-sass \
-  browser-sync \
-  chroma-sass \
-  del \
-  eslint \
-  event-stream \
-  gulp-autoprefixer \
-  gulp-eslint \
-  gulp-if \
-  gulp-load-plugins \
-  gulp-rename \
-  gulp-replace \
-  gulp-sass-lint \
-  gulp-size \
-  gulp-sourcemaps \
-  gulp.spritesmith \
-  kss \
-  node-sass-import-once \
-  sass-lint \
-  support-for \
-  typey \
-  zen-grids"
-
-RUN apk add --no-cache git \
-  && npm install -g $NPM_PACKAGES
+VOLUME /work
+WORKDIR /work
+
+RUN apk add --no-cache make g++ python git
+RUN apk add --no-cache curl && \
+  mkdir -p /opt && \
+  curl -sL https://yarnpkg.com/latest.tar.gz | tar xz -C /opt && \
```

Comment:
Seems like we already have `npm` here, so better to install yarn using npm instead